File: Mathematics/Count Number of Digit/CountNumberOfDigit.java
import java.io.*;
import java.util.*;
import static java.lang.System.out;

class CountNumberOfDigit {

    static int CountDigit(long number){
        int count = 0;
        while(number > 0){
            number = number/10;
            count++;
        }
        return count;
    }

    public static void main(String args[]){
        int number = 789;
        System.out.println(CountDigit(789));
    }
}

File: Mathematics/Count Number of Digit/CountNumberOfDigit.cpp
#include<iostream>
using namespace std;

int countDigit(int n){
    int result = 0;
    while(n > 0){
        n = n/10;
        result++;
    }
    return result;
}
int main()
{
    cout<<countDigit(12345)<<endl;
    return 0;
}